src/app/home/home.component.html

```diff
@@ -4,15 +4,16 @@
       <div class="card shadow border p-3">
         <div class="card-body">
           <h2 class="mb-3">What is "virtrolio"?</h2>
-          <p class="lead"><em>Virtrolio</em> stands for ‘virtual portfolio’ and is an online version of those pages in
+          <p class="lead"><em>Virtrolio</em> stands for ‘virtual folio’ and is an online version of those pages in
             your yearbook that you use to write messages to your friends. You can create your own virtrolio, share it
-            with friends, and have them write messages on it. Whenever you want, you can view your virtrolio or download
-            it.
+            with friends, and have them write customized messages on it. Whenever you want, you can view your virtrolio
+            and read all the wonderful messages you've received.
           </p>
-          <p class="lead" style="font-weight: 600">Tentative release: Mid-summer 2020.</p>
+          <p class="lead" style="font-weight: 600">Initial Release: Friday, July 3rd, 2020.</p>
           <p class="lead">Stay tuned and contact us at <a href="mailto:virtrolio.team@gmail.com" target="_blank">virtrolio.team@gmail.com</a>
-            if you have any questions.</p>
-          <img src="../../assets/images/logo_reg.png" alt="Virtrolio Logo" width="64" height="64">
+            if you have any questions. Also follow us at <b>@virtrolio</b> on Instagram and visit our Facebook page at <a
```

Comment:
Add link to instagram
```suggestion
            if you have any questions. Also follow us at <a href="https://www.instagram.com/virtrolio/" target="_blank">@virtrolio</a> on Instagram and visit our Facebook page at <a
```